_posts/2020-09-15-newpipe-0.20.0.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
+short: "NewPipe 0.20.0 released"
+date: 2020-09-15 05:00:00 +0100
+categories: [pinned, announcement, release]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Hey there! Guess who is back? We are. Admittedly, we're quite a bit late with this release. But there is a _very_ good reason for that. Are you ready for it? Sit down, grab some popcorn and a beverage of your choice. This is going to be long.
+<!-- more -->
+
+## A new unified player
+
+"What in the name of the coronavirus is the _unified player_?" you ask. Well, first of all, that's a very weird expression. Where did you even get that one from... - you know what? Save it. We don't want to know. You just keep on using that, we'll keep on talking about the new player.
+
+A new player it indeed is. Up until now, NewPipe had three different players bundled in its code: the main, pop-up and background players. All three had separate code and separate playing queues. When you switched from one player to another, the stream would start from the beginning instead of carrying on from where you left. In the background, the old player and its queue closed itself, while the new one took over. If that sounds complicated and cumbersome, that's because it is. At least if you ask us.
```

Comment:
```suggestion
A new player it indeed is. Until now, NewPipe had three different players bundled in its code: the main, pop-up and background players. All three had separate code and separate playing queues. When you switched from one player to another, the stream would start from the beginning instead of carrying on from where you left. In the background, the old player and its queue closed itself, while the new one took over. If that sounds complicated and cumbersome, that's because it is. At least if you ask us.
```